tests/actor/builtins/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "builtins-test-actor"
+version = "0.0.1"
+authors = [ "wasmcloud Team" ]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+async-trait = "0.1"
+console_log = "0.2"
+futures = "0.3"
+log = "0.4"
+serde_bytes = "0.11"
+serde_json ="1.0"
+serde = {version = "1.0", features = ["derive"]}
+regex = "1"
+
+wasmbus-rpc = "0.3"
+wasmcloud-interface-numbergen = { path = "../../../numbergen/rust" }
+wasmcloud-interface-logging = { path = "../../../logging/rust" }
+wasmcloud-test-util = "0.1"
+
+# comment out the following if this is inside a cargo workspace
+#[profile.release]
+## Optimize for small code size
+#lto = true
+#opt-level = "s"
```

Comment:
Do the compiler optimizations end up breaking the tests here? (I see your comment about the cargo workspace, but I wasn't sure if you got that fully configured). I'd vote to just remove these if it's not needed, or add them back if they are useful

Reply:
since they are the standard wasm flags, I figured I'd leave them, in case somebody copies the Cargo file.
They are commented out here because otherwise you get a warning about it being in a workspace. the workspace Cargo.toml has these settings uncommented.